index.html

```diff
@@ -67,6 +67,9 @@
 							Submit
 						</button>
 					</form>
+					<form class="navbar-form navbar-right" role="search">
+                        <select id="language" name="language" class="form-control"></select>
+                    </form>
```

Comment:
There's mixed whitespace (spaces/tabs) in these lines as well.

Reply:
It seems to me like someone should:
1. Close this PR
2. Open a new PR completely normalizing the whitespace in all files
3. After that is merged, open a new version of this PR